### ISSUE TITLE
Fix removing standard custom encoders and decoders

### ISSUE THREAD

**Issue by DarkVoyage** [gchudov/cuetools.net#264]
According to changelog this was fixed in 2.2.3. I've tried removing them again and they reappear again.
I've already configured needed external encoders and decoders and want to remove these "examples".
![image](https://user-images.githubusercontent.com/31579340/220377722-b6164205-122d-4ccc-b187-17fd344b6acb.png)

**Comment by c72578** [comment]
„Removing of manually added Encoders or Decoders“ has been fixed. See: #201

**Comment by c72578** [comment]
@DarkVoyage Thanks for your report.
The current behavior is unusual concerning deleting of predefined external Encoders or Decoders.
Two possibilities:
- Either gray out (disable) delete for predefined external Encoders or Decoders.
- Or allow deleting of predefined external Encoders or Decoders and do not let them reappear.

**Comment by DarkVoyage** [comment]
I don't think it is a big problem to allow removing them completely. If user wants to restore them he just can delete settings ini.
